packages/fxa-customs-server/README.md

```diff
@@ -6,9 +6,9 @@ This project is used by the [Firefox Accounts Auth Server](https://github.com/mo
 
 Clone the git repository and install dependencies:
```

Comment:
This section and the install memcached section can be removed because they are covered in the fxa-dev README.

Reply:
Done